client/client.go

```diff
@@ -18,6 +19,7 @@ type APIClient struct {
 	HostActivationAPI      *infra_provision.APIClient
 	InfraManagementAPI     *infra_mgmt.APIClient
 	KeysAPI                *keys.APIClient
+	FWAPI                  *fw.APIClient
```

Comment:
is this latest? I saw DFP as well in the generator template

Reply:
Yes this is latest , but as all the endpoints for DFP couldn't be tested , I have kept that changes in a separate branch.